### ISSUE TITLE
nosetests raises Runtime and Deprecation warnings

### ISSUE THREAD

**Issue by jkahn** [EducationalTestingService/rsmtool#29]
Seems like deprecation warnings should be disabled in testing if you're explicitly expecting them, or the tests should use something else.

But you're also getting a RuntimeWarning that might be meaningful -- and if not, worth trying to suppress, because it *looks* dire.
<details>
```
$ nosetests
./home/jeremy/Envs/rsmtool-dev/lib/python3.5/site-packages/numpy/lib/function_base.py:2079: RuntimeWarning: Degrees of freedom <= 0 for slice
  warnings.warn("Degrees of freedom <= 0 for slice", RuntimeWarning)
.................................................../home/jeremy/src/rsmtool/rsmtool/input.py:170: DeprecationWarning: The field name "expID" is deprecated and will be removed  in a future release, please use the new field name "experiment_id" instead.
  category=DeprecationWarning)
/home/jeremy/src/rsmtool/rsmtool/input.py:170: DeprecationWarning: The field name "train.lab" is deprecated and will be removed  in a future release, please use the new field name "train_label_column" instead.
  category=DeprecationWarning)
/home/jeremy/src/rsmtool/rsmtool/input.py:170: DeprecationWarning: The field name "LRmodel" is deprecated and will be removed  in a future release, please use the new field name "model" instead.
  category=DeprecationWarning)
/home/jeremy/src/rsmtool/rsmtool/input.py:170: DeprecationWarning: The field name "test" is deprecated and will be removed  in a future release, please use the new field name "test_file" instead.
  category=DeprecationWarning)
/home/jeremy/src/rsmtool/rsmtool/input.py:170: DeprecationWarning: The field name "test.lab" is deprecated and will be removed  in a future release, please use the new field name "test_label_column" instead.
  category=DeprecationWarning)
/home/jeremy/src/rsmtool/rsmtool/input.py:170: DeprecationWarning: The field name "trim.max" is deprecated and will be removed  in a future release, please use the new field name "trim_max" instead.
  category=DeprecationWarning)
/home/jeremy/src/rsmtool/rsmtool/input.py:170: DeprecationWarning: The field name "train" is deprecated and will be removed  in a future release, please use the new field name "train_file" instead.
  category=DeprecationWarning)
/home/jeremy/src/rsmtool/rsmtool/input.py:170: DeprecationWarning: The field name "trim.min" is deprecated and will be removed  in a future release, please use the new field name "trim_min" instead.
  category=DeprecationWarning)
/home/jeremy/src/rsmtool/rsmtool/input.py:170: DeprecationWarning: The field name "feature" is deprecated and will be removed  in a future release, please use the new field name "features" instead.
  category=DeprecationWarning)
/home/jeremy/src/rsmtool/rsmtool/input.py:205: DeprecationWarning: The model name "empWt" is deprecated and will be removed  in a future release, please use the new model name "LinearRegression" instead.
  category=DeprecationWarning)
..............................................................................................................................................................
----------------------------------------------------------------------
Ran 210 tests in 83.081s

OK

```

**Comment by desilinguist** [comment]
I was able to suppress the `RuntimeWarning` using a simple test setup fixture. 

However, the `DeprecationWarning` is a little tricker. I have explicitly set up `__init__.py`  such that `DeprecationWarnings` show up for the user even if they are running python with their own warning filters. This is done since we really want users to not use the legacy code. 

So, to filter out `DeprecationWarnings` in tests, I basically have to set an environment variable when running the nose command and then use the presence of this environment variable to the turn off the explicit filter in `__init__.py`.

**Comment by desilinguist** [comment]
Addressed by #34.
